Casks/bitcoin-core.rb

```diff
@@ -4,6 +4,7 @@
 
   url "https://bitcoincore.org/bin/bitcoin-core-#{version}/bitcoin-#{version}-osx.dmg"
   name "Bitcoin Core"
+  desc "Open-source, \"full\" Bitcoin client and wallet"
```

Comment:
```suggestion
  desc "Bitcoin client and wallet"
```
Better to keep the description as minimal as possible, I think. (You'll need to do `git pull --force` to pick up the changes from my rebase.)

Reply:
Yep, makes sense!